cmd/wait-for-github/pr_test.go

```diff
@@ -103,7 +133,7 @@ func TestPRCheck(t *testing.T) {
 				require.ErrorAs(t, err, &exitErr)
 				require.Equal(t, *tt.expectedExitCode, exitErr.ExitCode())
 			} else if err != nil {
-				require.ErrorAs(t, err, &tt.err)
+				require.NotNil(t, err)
```

Comment:
It would be nice to ensure the error we got is the same error we expect, but obviously this would add some complexity. Maybe add a todo for implementing later (if you think it'd be worth it).

Reply:
Good point - we don't have proper error types being returned from the code itself yet to be able to tell what's what, will add a TODO/issue to sort that out in future.